### DISCUSSION TITLE
更新0.4.19版本后，发现同步不了，提示需要https

### DISCUSSION THREAD

**Discussion by Wisees** [remotely-save/remotely-save#637] [Q&A]
更新0.4.19版本后，发现同步不了，提示需要https
[用的是群晖内网webdav，更新前0.4.19版本前可以正常使用，更新后提升需要https，群晖在内网使用，没有规划搞个https，大佬们帮忙看看能怎么解决呀
![IMG_4972](https://github.com/remotely-save/remotely-save/assets/13343268/5b4f507d-ee96-4849-a3dd-29beafcd1c82)

**Comment by fyears** [comment]
你的 Obsidian 版本和 iOS 版本是什么呢？这里主要是新版本 iOS 会限制，不是插件主动限制的呀

**Reply by Wisees** [reply]
手机上obsidian版本是1.5.12(134)，之前remotely save 的版本是0.3.40，手机是iPhone 14 pro max，iOS 17.4.1
iPad obsidian版本也是1.5.12(134)，remotely save的版本是0.3.40没有更新，现在iPad上使用还是正常的，iPadOS 17.4.1

**Comment by joy521125apple** [comment]
遇到了同样的问题，用的nas,没有https ;最好支持synolog drive ;

**Reply by fyears** [reply]
配置一下 https 吧。ios 限制无法解除

**Reply by guo0001jian** [reply]
我用的威联通nas，配置了https，在nas上建立的S3储存空间，用remotely save在windows电脑、安卓手机和ipad上同步笔记，实际情况是只有ipad上无法连接和同步。我再nas上建立webdav试试，结果也是只有ipad上无法连接和同步。请开发大大查看一下是不是IOS版的插件对https服务器的同步有问题，或者对SSL证书的验证方面有问题。
ipad系统是ios17.5.1，obsidian版本是1.6.5，remotely save版本为0.5.11.

**Reply by fyears** [reply]
错误提示截图是什么？证书是自签的还是合法的

**Reply by guo0001jian** [reply]
nas是威联通，证书是系统自带Let's Encryptq签发的合法的。S3服务其直接连不上，Error: Request failed, status 403。webdav服务器在检查连接时显示可以连接，但在点击同步按钮时会显示“Request failed. Could not connect tothe server.”，偶尔会显示“Request failed. An SSL error hasoccurred and a secure connection tothe server cannot be made.”
![4d5990f85387a08e5454e8e117b4997](https://github.com/remotely-save/remotely-save/assets/61703496/1cdb4218-ff0c-4859-85a7-ba25f1177414)

**Reply by fyears** [reply]
我建议先自测一下 https 在别的网站是否可用：

1. 新建一个测试 vault，随便建一些非空文件。
2. 注册一个 infinicloud 并尝试用 ios 连接和同步： https://github.com/remotely-save/remotely-save/blob/master/docs/remote_services/webdav_infinicloud_teracloud/README.md
3. 假如 infinicloud 没问题的话，那么显然就是你的 https 配置或者 webdav 配置有问题；假如 infinicloud 也有问题的话，那么可能就是我的插件或 obsidian 有问题。


此外，我的插件去掉了 https 的限制，可以试试直接连 http。注意，http 只适用于安全的内网，外网的话有严重的安全问题。

最大问题是我这里没有威联通硬件，如果是要复现的话，需要用虚拟机尝试复现，我也不懂你的具体操作（怎么配置 https 的）  https://sspai.com/post/78553
